pkg/cli/extension/get.go

```diff
@@ -0,0 +1,159 @@
+package extension
+
+import (
+	"os"
+	"strings"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/formatted"
+
+	"github.com/fuseml/fuseml-core/pkg/cli/client"
+	"github.com/fuseml/fuseml-core/pkg/cli/common"
+)
+
+const extensionGetTemplate = `{{decorate "bold" "ID"}}:	{{ .ID }}
```

Comment:
Wow, you really wrote a go template ... I'm not gonna read through this, so I'll just ask: is this a necessary way of doing it?

Reply:
I wouldn't say it's necessary, but it does makes it easier to read :)
we do the same for `workflow get`, I just borrowed the idea

Reply:
_easier to read_ in combination of _golang templates_? That's something...